tests/e2e/standalone/standalone_test.go

```diff
@@ -243,12 +245,15 @@ func testInstall(t *testing.T) {
 				return
 			}
 			// Check version
-			output, err := spawn.Command(file, "--version")
+			output, err := spawn.Command(file, "--version", "2>/dev/null")
 			if !assert.NoError(t, err) {
 				return
 			}
 			output = strings.TrimSpace(output)
-			if !assert.Equal(t, version, output) {
+			// Changing version check since there is a log that is output on daprd --version
+			// 2021/11/12 11:10:38 maxprocs: Leaving GOMAXPROCS=12: CPU quota undefined
+			// before the version is output
+			if !assert.Contains(t, output, version) {
```

Comment:
Do we need both "2>/dev/null" above and assert.Contains? Should only need one or the other?

Reply:
```suggestion
			if !assert.Equal(t, output, version) {
```